projects/aws/upgrader/cmd/root.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/aws/eks-anywhere-build-tooling/tools/version-tracker/pkg/util/logger"
```

Comment:
Was discussing with @abhay-krishna. Can we move this logger to `eks-anywhere-build-tooling/pkg` and refactor this and version-tracker projects to use that?